packages/twenty-front/src/modules/ui/navigation/menu-item/internals/components/MenuItemLeftContent.tsx

```diff
@@ -14,7 +14,7 @@ type MenuItemLeftContentProps = {
   className?: string;
   LeftIcon: IconComponent | null | undefined;
   showGrip?: boolean;
-  text: string;
+  text?: string;
```

Comment:
What does it mean to have a MenuItem without content?

Reply:
Should i make the text equal to an empty string? I thought text was not needed when creating the right content of the menu item as it's only an icon (line 70 in MenuItem.tsx)

Reply:
see MenuItemSelect.tsx to see how to add a Right icon and implement it in MenuItem direclty. Do not MenuItemLeftContent